<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cashier_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cashier </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ASSETS/cashier+icon-1320184411872118482_128.png</normaloff>ASSETS/cashier+icon-1320184411872118482_128.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>92</width>
     <height>92</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="info_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Info    </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ASSETS/info-131964752893297302_128.png</normaloff>ASSETS/info-131964752893297302_128.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="database_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ASSETS/officedatabase_103574.png</normaloff>ASSETS/officedatabase_103574.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Admin   </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ASSETS/pngfind.com-admin-icon-png-5286002.png</normaloff>ASSETS/pngfind.com-admin-icon-png-5286002.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>95</width>
     <height>95</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="cashier_win" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
   <widget class="QPushButton" name="pizza_button">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pizza</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/pizza/pepporoni pizza.png</normaloff>ASSETS/pizza/pepporoni pizza.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sandwich_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sandwich</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/sandwiches/burger.png</normaloff>ASSETS/sandwiches/burger.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="drinks_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>drinks</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/drinks/pepsi can.png</normaloff>ASSETS/drinks/pepsi can.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="salad_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>salad</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/salad/salad.png</normaloff>ASSETS/salad/salad.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deserts_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deserts</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/deserts/ice cream.png</normaloff>ASSETS/deserts/ice cream.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fries_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fries</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ASSETS/fries/fries.png</normaloff>ASSETS/fries/fries.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>26</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>142</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>23</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>29</number>
    </attribute>
    <column>
     <property name="text">
      <string>item</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>price</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="total_price">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total_pric">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>0$</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="cashier_button_win" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="info_button_win" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="admin_button_win" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="database_button_win" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="info_win" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="database_win" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <widget class="QWidget" name="admin_win" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 45, 53);</string>
   </property>
  </widget>
  <zorder>admin_win</zorder>
  <zorder>info_win</zorder>
  <zorder>database_win</zorder>
  <zorder>admin_button_win</zorder>
  <zorder>database_button_win</zorder>
  <zorder>info_button_win</zorder>
  <zorder>cashier_button_win</zorder>
  <zorder>cashier_button</zorder>
  <zorder>info_button</zorder>
  <zorder>database_button</zorder>
  <zorder>admin_button</zorder>
  <zorder>cashier_win</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
